.github/workflows/release.yml

```
@@ -0,0 +1,36 @@
+  push:
+    branches:
+      - release  # Trigger when pushing to the main branch. Adjust this to match your default branch.
+  workflow_dispatch:  # Allow manual triggering of the workflow from GitHub UI
+
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)